targets/targets.json

```diff
@@ -2042,113 +2042,6 @@
         "device_has_remove": ["QSPI"],
         "release_versions": ["2", "5"]
     },
-    "LPC55S69": {
```

Comment:
Commit "Remove v8-M and dual v7-M targets" seems only to remove v8-M targets. Consider changing commit message.

Reply:
Yeah, rebasing on top master had removed dual v7-M targets. Updated the commit message.